tls/extensions/s2n_client_key_share.c

```diff
@@ -253,10 +265,8 @@ uint32_t s2n_extensions_client_key_share_size(struct s2n_connection *conn)
             + S2N_SIZE_OF_EXTENSION_DATA_SIZE
             + S2N_SIZE_OF_CLIENT_SHARES_SIZE;
 
-    for (uint32_t i = 0; i < ecc_pref->count ; i++) {
-        s2n_client_key_share_extension_size += S2N_SIZE_OF_KEY_SHARE_SIZE + S2N_SIZE_OF_NAMED_GROUP;
-        s2n_client_key_share_extension_size += ecc_pref->ecc_curves[i]->share_size;
-    }
+    s2n_client_key_share_extension_size += S2N_SIZE_OF_KEY_SHARE_SIZE + S2N_SIZE_OF_NAMED_GROUP;
```

Comment:
I thought the extension refactor was completed already? I didn't think this function still existed.

Reply:
Even though the extension refactor is complete the functions are not yet completely removed from unit testing. I had to modify this portion to prevent breaking tests.